src/amr/solvers/solver_ppc.hpp

```diff
@@ -467,10 +481,11 @@ void SolverPPC<HybridModel, AMR_Types>::corrector_(level_t& level, HybridModel&
             resourcesManager->setTime(B, *patch, newTime);
         }
 
-        fromCoarser.fillMagneticGhosts(B, levelNumber, newTime);
+        std::cout << "faraday done on level " << levelNumber << "\n";
+        // fromCoarser.fillMagneticGhosts(B, levelNumber, newTime);
     }
 
-
+    // TODO misses AMPERE here
```

Comment:
should this be done before merging?

Reply:
probably, it's weirdly been missing since the beginning....

Reply:
have created issue #742 to deal with it in isolation